lib/tasks/create_site.rake

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+  
+# Tasks to add default site code of 'sfsg' to all existing resources and categories
+
+namespace :create_site do
+  # create site_code of 'sfsg'
+  desc 'Creates a site for sfsg'
+  task add_sfsg_site: :environment do
+    Site.transaction do
+      Site.find_or_create_by(site_code: 'sfsg')
+    end
+  end
+  
+  # to each resource, add a connection using resource_site
+  desc 'Adds the sfsg site to existing resources'
```

Comment:
We might need to refine the business logic a bit here, as I think (whoops) some of the sffamilies orgs have already been added. We can do that in another PR though.